docs/variables.md

```diff
@@ -1488,11 +1514,43 @@ The variable was implemented in [Percona Server for MySQL 8.0.20-11](release-not
 | Data type    | Boolean                          |
 | Default      | OFF                              |
 
+#### Version changes
+
 The variable was implemented in [Percona Server for MySQL 8.0.25-15](release-notes/Percona-Server-8.0.25-15.md#id1).
 
-DBOptions::enable_pipelined_write for RocksDB.
+#### Improving Write Throughput with Pipelined Writes
+
+This option maps directly to RocksDB’s `DBOptions::enable_pipelined_write`. For details, see the [RocksDB documentation on Pipelined Write](https://github.com/facebook/rocksdb/wiki/Pipelined-Write).
+
+
+RocksDB uses pipelined writes to increase throughput in workloads with  
+concurrent writers. When the Write-Ahead Log (WAL) is enabled, write  
+operations normally pass through a shared queue. Each writer appends to the  
+WAL and then updates the memtable in strict sequence. This pattern limits  
+parallelism.
+
+With pipelined writes, RocksDB overlaps the WAL and memtable stages across  
+writers. As soon as one thread completes its WAL write, the next thread in  
+line can begin its own WAL operation—even if the previous thread has not yet  
+updated the memtable. This change interleaves write stages, reduces latency,  
+and improves overall write throughput.
+
+Enabling `rocksdb_enable_pipelined_write` activates this feature. It is  
+especially useful in workloads with high concurrency or transaction phases  
+that generate bursts of write activity, such as the prepare step in  
+two-phase commit protocols.
```

Comment:
Original text: beneficial in 
a) workloads with multiple concurrent writers [and]
b) in scenarios like the prepare phase of a two-phase commit

"prepare phase" should stay because it's a proper name:
Two-Phase Commit (2PC) is a distributed algorithm used to ensure atomicity across multiple systems or databases in a transaction. It ensures that either all participants commit the transaction or none of them do, preventing data inconsistency.
**Phase** 1: **Prepare** (Voting Phase)
Phase 2: Commit (Decision Phase)